src/styles/aligns.ts

```diff
@@ -0,0 +1,7 @@
+export const aligns = {
```

Comment:
I think you don't need two exports here, remove this one

Reply:
Updated for all style files